src/test/java/cloud/eppo/helpers/AssignmentTestCaseDeserializer.java

```diff
@@ -0,0 +1,76 @@
+package cloud.eppo.helpers;
+
+import cloud.eppo.ufc.dto.EppoValue;
+import cloud.eppo.ufc.dto.SubjectAttributes;
+import cloud.eppo.ufc.dto.VariationType;
+import cloud.eppo.ufc.dto.adapters.EppoValueDeserializer;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONException;
+
+public class AssignmentTestCaseDeserializer extends StdDeserializer<AssignmentTestCase> {
```

Comment:
Mostly not new code, ported over from Android implementation ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/androidTest/java/cloud/eppo/android/helpers/AssignmentTestCaseDeserializer.java))